README.MD

```diff
@@ -67,6 +67,10 @@ influx_db:
             http_port:            8086
             username:             ''
             password:             ''
+
+            # Setup timeout or connection timeout (seconds) for your requests
```

Comment:
This should be also on the `info()` method of the configuration.

Please add this. The best thing to do for the documentation is to use `bin/console config:dump-reference` and copy/paste the result.